src/hearings/components/multi-level-selector/multi-level-selector.component.ts

```diff
@@ -1,30 +1,64 @@
-import { Component, Input } from '@angular/core';
-import { FormArray, FormBuilder, FormGroup } from '@angular/forms';
+import { AfterViewInit, Component, Input } from '@angular/core';
+import { AbstractControl, FormArray, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ControlTypeEnum } from '../../models/hearings.enum';
 
 @Component({
   selector: 'exui-multi-level-selector',
   templateUrl: './multi-level-selector.component.html',
   styleUrls: ['./multi-level-selector.component.scss']
 })
-export class MultiLevelSelectorComponent {
+export class MultiLevelSelectorComponent implements AfterViewInit {
   @Input() public multiLevelSelect: FormArray;
   @Input() public configLevels: { level: number, controlType: ControlTypeEnum }[];
+  @Input() public hasValidationRequested: boolean = false;
   @Input() public level: number = 1;
   public formGroup: FormGroup;
   constructor(public fb: FormBuilder) {
-    this.formGroup = fb.group({ item: [''] });
+    this.formGroup = fb.group({ item: ['', Validators.required] });
     this.formGroup.controls.item.valueChanges.subscribe(value => {
+      (this.multiLevelSelect as FormArray).controls.forEach(x => {
+        x.value.selected = false;
+      });
       (this.multiLevelSelect as FormArray).controls.filter(x => x.value.key === value).forEach(x => {
         x.value.selected = true;
       });
     });
   }
 
+  ngAfterViewInit(): void {
+    this.assignSelectedOptionToItemControl();
+  }
+
+  public assignSelectedOptionToItemControl() {
+    if (this.controlType === ControlTypeEnum.SELECT) {
+      console.log('length controls', (this.multiLevelSelect as FormArray).controls.length);
+      console.log('filtered controls', (this.multiLevelSelect as FormArray).controls
+        .filter(control => control.value && control.value.selected === true).length);
+      (this.multiLevelSelect as FormArray).controls
+        .filter(control => control.value && control.value.selected === true)
```

Comment:
I don't think we need to explicitly check `control.value.selected === true` it could be `control.value.selected`